packages/next/src/shared/lib/stack-trace-utils.ts

```diff
@@ -0,0 +1,6 @@
+// Hide the method name when showing a stack trace in the error overlay or when displaying a stack trace in the console.
+export function isHiddenMethodName(methodName: string) {
+  return /^(?:Object\.|Module\.)?(?:<anonymous>|eval|__TURBOPACK__module__evaluation__)$/.test(
+    methodName
+  )
```

Comment:
I don't understand why we would hide these names. `eval` and `<anonymous>` I can maybe see but that should apply everywhere. The rest should be sourcemapped names.